src/app/githook/templates/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Run ESLint on staged files
+eslint --ignore-path .gitignore  "**/*.{js,jsx}"
```

Comment:
Could this be adapted to run the `npm` command instead?

```suggestion
npm run code-config:lint
```

Reply:
havent tested it but I dont see any issues with it

Reply:
you may need to update the part where the `code-config:lint` script is injected to be more like this command